src/test/java/StringCalculatorTest.java

```diff
@@ -0,0 +1,73 @@
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class StringCalculatorTest {
+
+  private StringCalculator calculator;
+
+  @BeforeEach
+  private void init() {
+    calculator = new StringCalculator();
+  }
+
+  @Test
+  @DisplayName("빈 문자열 또는 null값을 입력할 경우 0을 반환해야 한다.")
+  void zeroReturn() {
+    int expect = 0;
+    assertThat(calculator.calculate("")).isEqualTo(expect);
+    assertThat(calculator.calculate(null)).isEqualTo(expect);
```

Comment:
```suggestion
    assertThat(calculator.calculate("")).isEqualTo(0);
    assertThat(calculator.calculate(null)).isEqualTo(0);
```

굳이 로컬 변수롤 빼지 말고 0을 바로 쓰는 것은 어떨까?